main.py

```diff
@@ -0,0 +1,116 @@
+#Amazon data set VADER analysis
+
+import pandas as pd

```

Comment:
create a requirements.txt file with each library you use and its version, so that I can easily install the dependencies (pandas, numpy, ...)

Reply:
fixed